sklearn/metrics/tests/test_classification.py

```diff
@@ -1924,8 +1924,10 @@ def test_log_loss():
 
     # check eps and handling of absolute zero and one probabilities
     y_pred = np.asarray(y_pred) > .5
-    loss = log_loss(y_true, y_pred, normalize=True, eps=.1)
-    assert_almost_equal(loss, log_loss(y_true, np.clip(y_pred, .1, .9)))
```

Comment:
I can't think why there is any need to use such a high epsilon here. In the case someone did want to, I don't think they would expect the same result. I think this test is just to make sure the functionality is working anyway, so no harm in changing it to a more normal level.